tests/trx_generator/main.cpp

```diff
@@ -28,7 +28,7 @@ int main(int argc, char** argv) {
    et::accounts_config accts_config;
    et::trx_tps_tester_config tester_config;
 
-   const int64_t trx_expiration_max = 3600;
+   const int64_t trx_expiration_max = 3599;
```

Comment:
why 3599?

Reply:
The default on-chain limit is 3600 and the comparison is `<= pending_block_time`. This should prevent someone specifying 3600 and hitting that limit. Although, since that is an on-chain configurable value, maybe `trx_generator` should not have any hard-coded max.